packages/secretlint/test/create-secretlintrc.test.ts

```diff
@@ -0,0 +1,67 @@
+// LICENSE : MIT
+"use strict";
+import assert from "assert";
+import path from "path";
+import fs from "fs";
```

Comment:
```suggestion
import fs from "fs/promises";
```

We prefer to use `fs/promises` that provide [Promises API](https://nodejs.org/api/fs.html#promises-api).